pkgs/top-level/all-packages.nix

```diff
@@ -10223,6 +10223,8 @@ in
 
   riak = callPackage ../servers/nosql/riak/2.1.1.nix { };
 
+  stanchion = callPackage ../servers/nosql/riak-cs/stanchion.nix { };
```

Comment:
Is it, though? Now that `erlang_basho_R16B03` is in the main repo, it doesn't make much sense to use anything but that. Also, this package doesn't work with any interpreter but R16|R17. What if a user passes in a faulty interpreter?